bib/cmd/bootc-image-builder/image.go

```diff
@@ -209,6 +209,18 @@ func genPartitionTable(c *ManifestConfig, customizations *blueprint.Customizatio
 	if err != nil {
 		return nil, fmt.Errorf("error reading disk customizations: %w", err)
 	}
+
+	// Embedded disk customization applies if there was no local customization
+	if fsCust == nil && diskCust == nil && c.SourceInfo != nil && c.SourceInfo.ImageCustomization != nil {
+		imageCustomizations := c.SourceInfo.ImageCustomization
+
+		fsCust = imageCustomizations.GetFilesystems()
```

Comment:
Having a test for this would be really good, I hesitate to ask you to write as it might be a bit annoying/involved - I am happy to look into it in the morning and provide a diff.

Reply:
added unit tests